integ-test/src/test/java/org/opensearch/sql/sql/NestedPartiQLIT.java

```diff
@@ -0,0 +1,50 @@
+package org.opensearch.sql.sql;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.opensearch.sql.legacy.SQLIntegTestCase;
+
+import java.io.IOException;
+import java.util.List;
```

Comment:
Do these imports pass checkstyle with the extra space and them out of alphabetical order?